### PR TITLE
add t3js v1.4.1 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/t3js/package.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "component"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/box/t3js.git",
+    "basePath": "dist",
+    "files": [
+      "t3.min.js",
+      "t3.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/t3js/package.json]: Should at least also add `t3.min.js.map`.
I think we also need `t3-native.js`, `t3-jquery` related files.

**Comment by maruilian11** [comment]
problems are fixed, please take a look, thx

File: ajax/libs/t3js/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    "client-side",
+    "modular",
+    "component"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/box/t3js.git",
+    "basePath": "dist",
+    "files": [
+      "*"
```
Comment (PeterDaveHello) [inline ajax/libs/t3js/package.json]: Should not use wildcard in this case.
https://github.com/cdnjs/cdnjs#conventions
> Filenames should not include a version number and be lowercase.
> This is OK: useful.min.js, but this is not: useful-2.0.1.min.js.

**Comment by maruilian11** [comment]
fixed

**Comment by PeterDaveHello** [comment]
t3.min.js has src, but t3-native.min.js and t3-jquery.min.js don't, let's include them to make it consistent.

**Comment by maruilian11** [comment]
fixed again.

**Comment by PeterDaveHello** [comment]
same same

**Comment by PeterDaveHello** [comment]
any updates?

**Comment by maruilian11** [comment]
yes, plz take a look again.

**Comment by PeterDaveHello** [comment]
Commit's parent was too old, please rebase on our latest master branch.

**Comment by maruilian11** [comment]
done.

**Pull request merged**